tests/pipeline/test_pipeline_load_info.py

```diff
@@ -0,0 +1,26 @@
+import dlt
+
+
+data = [
+    {"id": 1, "name": "Alice"},
+    {"id": 2, "name": "Bob"},
+]
+
+
+def test_pipeline_load_info_metrics_schema_is_not_chaning() -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="quick_start",
+        destination="duckdb",
+        dataset_name="mydata",
+    )
+
+    load_info = pipeline.run(data, table_name="users")
+
+    pipeline.run([load_info], table_name="_load_info")
```

Comment:
1. please add load_info, normalize_info and extract_info.
2. please load to a separate schema (you have schema argument to run()
3. use this second schema to compare hashes (mind that it wont be a default)